modules/cluster-logging-updating-elasticsearch.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-elasticsearch_{context}"]
+= Updating Elasticsearch
+
+Updating cluster logging to 4.5 also upgrades your cluster to use Elasticsearch 6. After updating cluster logging to 4.5,You must perform the following tasks to configure your cluster to work with the new version of Elasticsearch.
+
+.Prerequisites
+
+* Update cluster logging from 4.5 to 4.6.
+
+* Make sure the cluster logging status is healthy:
++
+** All Pods are `ready`.
+** Elasticsearch cluster is healthy.
+
+* Take a snapshot of your Elasticsearch and Kibana indices. For more information, see link:https://www.elastic.co/guide/en/elasticsearch/reference/7.6/snapshot-restore.html[Snapshot and restore] in the Elasticseach documentation.
```

Comment:
In line with my other comment, I don't believe we want to ref the snapshot api at all.  In the past we have simply suggested to make a backup of PVCs and in the context of AWS that is also a "snapshot" but not the same as ES snapshot